src/stylesheets/components/current-task/currentTask.scss

```diff
@@ -0,0 +1,34 @@
+.current-task-container {
+	width: 100%;
+	height: 100%;
+}
+
+.current-task-title {
+	width: 100%;
+	text-align: left;
+	font-size: 1.5rem;
+}
+
+.current-task-content-container {
+	width: 100%;
+	height: calc(100% - 48px);
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	padding: 24px 16px;
+
+	background: linear-gradient(
+		148.39deg,
+		rgba(39, 127, 242, 0.48) 0%,
+		rgba(39, 127, 242, 0.12) 100%
+	);
+	backdrop-filter: blur(100px);
```

Comment:
Is this necessary? I dont think there's anything behind the current task container, so this won't really be doing anything